core/go/internal/publictxmgr/in_flight_transaction_stage_controller.go

```diff
@@ -315,7 +315,8 @@ func (it *inFlightTransactionStageController) ProduceLatestInFlightStageContext(
 									rsc.InMemoryTx.GetGasPriceObject()
 									gasPriceJSON, _ := json.Marshal(rsc.InMemoryTx.GetGasPriceObject())
 									rsc.StageOutputsToBePersisted.TxUpdates.NewSubmission = &DBPubTxnSubmission{
-										SignerNonce:     rsc.InMemoryTx.GetSignerNonce(),
+										from:            rsc.InMemoryTx.GetFrom().String(),
```

Comment:
This is just so we have a key to use to dispatch all submissions for the same `from` address, to the same writer (as the submission itself doesn't have a from address in it)